src/mesh/NodeDB.cpp

```diff
@@ -56,10 +56,6 @@
 #include <WiFiOTA.h>
 #endif
 
-// stringify
-#define xstr(s) str(s)
-#define str(s) #s
-
```

Comment:
Did this get moved somewhere? This is needed to use the build epoch as a filename.

Reply:
Ah, it's already in configuration.h.